images/build/Dockerfile

```diff
@@ -277,9 +277,9 @@ RUN apt-get update \
 # https://github.com/docker-library/python/issues/147
 ENV PYTHONIOENCODING UTF-8
 COPY build/__python-versions.sh /tmp
-COPY --from=mcr.microsoft.com/oryx/python-build-base:2.7-20190530.1 /opt /opt
-COPY --from=mcr.microsoft.com/oryx/python-build-base:3.6-20190530.1 /opt /opt
-COPY --from=mcr.microsoft.com/oryx/python-build-base:3.7-20190530.1 /opt /opt
+COPY --from=mcr.microsoft.com/oryx/python-build-2.7:20190402.1 /opt /opt
```

Comment:
@arryoc I reverted this change of yours because it is failing the Shapely integration tests. Please investigate why they fail on moving to the new tags.

Reply:
Ignore my comment. Seems like the error is happening in runtime image of Python

Reply:
@gpcastro FYI..this could be a regression from your changes to Python runtime images having similar structure as the build image. Could you investigate it?

Reply:
@kichalla I think we need to add the virtualenv command back to this test … commit beb4961a
I will send a PR, please take a look

Reply:
@kichalla I think we need to remove this test ( as I see another test for shapely is running correctly). Another thing  I noticed the path in run.sh is wrong export PYTHONPATH=$PYTHONPATH:"$virtualEnvDir/lib/python$PYTHON_VERSION/site-packages", it should be export PYTHONPATH=$PYTHONPATH:"$virtualEnvDir/opt/python$PYTHON_VERSION/site-packages"
looking more into it

Reply:
as per the current logic in the python script generator, we assume virtualenv is used whenever there is manifest file, but when we use packagedir we are not using virtualenv though manifest file is still getting generated which has operation_id. so it's throwing error. When I deleted manifest file, it worked for me. This seems to be a bug

Reply:
bug 903955 is already created it seems

Reply:
The bug is with Python startup script IMO as it needs to check if virtual environment was used or not correctly. I am fixing it and will send out an update.

Reply:
Can you explain what the issue is? there's too many changes it's hard to find out.

Reply:
The issue was that Python startup script was only considering virtualenv if a manifest is just present on disk. We had few tests which use package directory which were failing due to it. This issue came to light because of my change related to OperationId. Since we always write OperationId to manifest file now, we are going to have this file created always which caused this issue to pop-up.